package.json

```diff
@@ -7,11 +7,13 @@
     "@testing-library/react": "^9.3.2",
     "@testing-library/user-event": "^7.1.2",
     "axios": ">=0.21.1",
+    "bootstrap": "^4.6.0",
     "bootstrap-4-react": "0.0.59",
     "bootstrap4": "0.0.1-security",
     "concurrently": "^6.0.0",
     "nodemailer": "^6.4.11",
     "react": "^16.13.1",
+    "react-bootstrap": "^1.5.2",
```

Comment:
We have used `bootstrap-4-react` in this project so could you mind using that only for adding the spinner. Adding `bootstrap` and `react-bootstrap` don't make sense IMO.
Also make the spinner to `center`.

Reply:
Alright!

Reply:
@Namanl2001 `bootstrap-4-react`  doesn't have css files, does it? Can I know how to import spinner if possible?

Reply:
@sagar-barapatre / @guptaji6489  Could help you.

Reply:
Have a look [here](https://react-bootstrap.github.io/components/spinners/)

Reply:
@sagar-barapatre , I mean there are no css files in node_modules/bootstrap-4-react